drone-fly-app/src/main/java/com/expediagroup/dataplatform/dronefly/app/service/factory/ListenerCatalogFactory.java

```diff
@@ -36,6 +37,15 @@ public ListenerCatalog newInstance(String confProvidedList) {
       log.info("{apiary.listener.list} is empty. Going to look in hive-site.xml if it is provided on the classpath.");
       listenerImplList = hiveConf.getVar(HiveConf.ConfVars.METASTORE_EVENT_LISTENERS);
     }
+
+    if (StringUtils.isBlank(listenerImplList)) {
+      log
+          .warn(
+              "No Hive metastore listeners have been provided as argument {apiary.listener.list} or hive-site.xml. Going to use: {}",
+              LoggingMetastoreListener.class.getName());
+      listenerImplList = LoggingMetastoreListener.class.getName();
```

Comment:
I think it's a little bit clearer if you put this on line 42 and then in the log message you can just put `listenerImplList` instead of calling `class.getName()` twice.